notebooks/250-efficient-object-detection/250-efficient-object-detection.ipynb

```diff
@@ -0,0 +1,622 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    model_ir = core.read_model(model=XML_PATH)
</pre><p>how do you decide when need to use named parameters and when not? </p><p>in read model line you use it, however only one parameter provided, but in compile_model not, however multiple parameters provided. Could you please align here and cell above?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/871/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>